src/rebar_packages.erl

```diff
@@ -128,7 +128,7 @@ registry_checksum({pkg, Name, Vsn, _Hash}, State) ->
         ets:lookup_element(?PACKAGE_TABLE, {Name, Vsn}, 3)
     catch
         _:_ ->
-            throw(?PRV_ERROR({missing_package, ec_cnv:to_binary(Name), ec_cnv:to_binary(Vsn)}))
```

Comment:
In that case I'd go for an explicit 3-arity for this one and keep them separate, personally. If anybody else or some other component handled that exception before, we won't accidentally break their code, absent any other criteria.